### ISSUE TITLE
SDWebImage/WebP pod spec needs updated due to libwebp's spec bumping to 1.0

### ISSUE THREAD

**Issue by mthole** [SDWebImage/SDWebImage#2431]
SDWebImage/WebP depends on libwebp, which recently had it's pod spec update to 1.0.0:
https://github.com/CocoaPods/Specs/commit/e47e76226124c705f6205854cc6d6b7ee9154441

This major version bump to 1.0.0 (despite pointing to the same v0.6.1 source tag!) causes SDWebImage/WebP's dependency matcher with version `~> 0.5` to fail.

In my case, this caused `pod update` to actually *downgrade* SDWebImage in order to update libwebp, which was not the desired behavior.

If SDWebImage/WebP updates it's dependency to `~> 1.0` and pushes a new spec version, this should resolve this error and prevent it from happening for others.

**Comment by bpoplauschi** [comment]
@mthole very good catch. First of all, I made a mistake with publishing the 1.0.0 release, as `"tag": "v0.6.1"`. I will correct that.
I think that is causing the inconsistency, because otherwise CocoaPods should just know SDWebImage supports only 0.x versions and not upgrade to 1.0.0

**Comment by bpoplauschi** [comment]
I created https://github.com/CocoaPods/Specs/pull/14381 to fix the issue

**Comment by mthole** [comment]
I solved this for myself by removing a separate `pod 'libwebp'` in my Podfile, which wasn't necessary anyway. If you run into this issue, try that!

**Comment by bpoplauschi** [comment]
Great, thanks @mthole for sharing. I will still push to fix the latest `libwebp` `podspec`.

**Comment by bpoplauschi** [comment]
I will close this issue now

**Comment by dreampiggy** [comment]
@bpoplauschi 😅Seems that libwebp pod trunk owner is not you. Need you try to connect the owner and delete that v1.0.0 version and fix it ?

**Comment by bpoplauschi** [comment]
@dreampiggy I am one of the owners of `pod trunk` `libwebp`. 
```
pod trunk info libwebp

libwebp
    - Versions:
      - 0.3.0-rc7 (2014-05-19 21:52:10 UTC)
      - 0.3.1 (2014-05-19 21:59:26 UTC)
      - 0.4.0 (2014-05-19 22:02:33 UTC)
      - 0.4.1 (2014-08-08 15:06:06 UTC)
      - 0.4.2 (2014-11-04 08:59:32 UTC)
      - 0.4.3 (2015-03-17 13:26:28 UTC)
      - 0.4.4 (2016-01-18 15:10:52 UTC)
      - 0.5.0 (2016-01-18 15:15:46 UTC)
      - 0.5.1 (2016-09-29 15:21:39 UTC)
      - 0.5.2 (2017-05-05 19:01:52 UTC)
      - 0.6.0 (2017-05-05 19:15:29 UTC)
      - 0.6.1 (2018-04-04 08:07:07 UTC)
      - 1.0.0 (2018-07-31 07:47:06 UTC)
    - Owners:
      - Bogdan Poplauschi <bpoplauschi@gmail.com>
      - Shahar Hadas <shaharhd@gmail.com>
```
But I don't know how to delete and resubmit. That is why I created a PR to the Specs repo. I'm waiting for someone on their team to advise.

**Comment by bpoplauschi** [comment]
Actually, now I saw they added `pod trunk delete`

**Comment by dreampiggy** [comment]
@bpoplauschi What about `pod trunk delete libwebp 1.0.0` ? What's the output ?

**Comment by bpoplauschi** [comment]
I just tried it and it worked. :) cool. This option was not always around, or at least I didn't see it.
Uploading the correct 1.0.0 version. Thanks @dreampiggy

**Comment by bpoplauschi** [comment]
Corrected version 1.0.0 is published.
@mthole we will update our dependencies to that, probably for version 5.0. But if you are using an older version of SDWebImage, it will pick up the compatible version of libwebp - in fact, this is CocoaPods's job.
